libs/community/langchain_community/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -0,0 +1,276 @@
+import uuid
+from typing import Any, Dict, Iterable, List, Optional, Tuple, Type
+
+import numpy as np
+from azure.cosmos.container import ContainerProxy
+from azure.cosmos.cosmos_client import CosmosClient
+from azure.cosmos.database import DatabaseProxy
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VST, VectorStore
+
+from langchain_community.vectorstores.utils import maximal_marginal_relevance
+
+
+# You can read more about vector search using AzureCosmosDBNoSQL here.
+# https://aka.ms/CosmosVectorSearch
```

Comment:
just redirects to https://www.microsoft.com/en-us/ for me

Reply:
It should work now, as it is has now been announced publicly at [Microsoft Buiild](https://register.build.microsoft.com/ )